test/utils/hooks.ts

```diff
@@ -14,12 +21,24 @@ declare global {
 
   // eslint-disable-next-line @typescript-eslint/no-namespace
   namespace WebdriverIO {
-    interface Browser {
+    interface Browser
+      extends WebdriverIOQueries,
+        WebdriverIOQueriesChainable<Browser> {
       openKeeperPopup: () => Promise<void>;
     }
+
+    interface Element
+      extends WebdriverIOQueries,
+        WebdriverIOQueriesChainable<Element> {}
   }
 }
 
+declare module 'webdriverio' {
+  // eslint-disable-next-line
```

Comment:
It's a good idea to only disable a particular rule, instead of all like you did in that case). You need to append rule name for that. You can find a lot of examples of there throughout a codebase)

Reply:
Done